sdk/go/common/workspace/project.go

```diff
@@ -63,14 +63,18 @@ type Analyzers []tokens.QName
 
 // ProjectTemplate is a Pulumi project template manifest.
 type ProjectTemplate struct {
+	// DisplayName is an optional user friendly name of the template.
+	DisplayName string `json:"displayName,omitempty" yaml:"displayName,omitempty"`
 	// Description is an optional description of the template.
 	Description string `json:"description,omitempty" yaml:"description,omitempty"`
 	// Quickstart contains optional text to be displayed after template creation.
 	Quickstart string `json:"quickstart,omitempty" yaml:"quickstart,omitempty"`
 	// Config is an optional template config.
 	Config map[string]ProjectTemplateConfigValue `json:"config,omitempty" yaml:"config,omitempty"`
-	// Important indicates the template is important and should be listed by default.
```

Comment:
quick note on this change - it looks like this used to be the case but is no longer true since [templatesToOptionArrayAndMap](https://github.com/pulumi/pulumi/blob/master/pkg/cmd/pulumi/new.go#L1113) is always called with `showAll` as true